dubbo-spring-boot/dubbo-spring-boot-observability-starter/pom.xml

```diff
@@ -94,6 +94,12 @@
             <artifactId>dubbo-spring-boot-starter</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-common</artifactId>
+            <version>${project.version}</version>
+            <optional>true</optional>
+        </dependency>
```

Comment:
After introducing dubbo-spring-boot-starter, is there no need to introduce the common package separately?

Reply:
when I remove `dubbo-common`, it occurs:
![image](https://user-images.githubusercontent.com/56248584/227948139-4e0c2b1a-336e-41e7-9d3a-43c5a8950099.png)